src/components/ReviewsContainer/ListOfReviews.jsx

```diff
@@ -0,0 +1,12 @@
+import Review from "./Review"
+
+const ListOfReviews = ({reviews}) => {
+    return (
+        <div>
```

Comment:
this is currently a div but I feel its only because react is forcing you to group these in a parent? we can use <></> in react if this is the only reason for the div